data/data/coreos/rhcos.json

```diff
@@ -1,107 +1,107 @@
 {
   "stream": "rhcos-4.15",
```

Comment:
@mike-nguyen what updates this to be 4.16?

Reply:
Hmm good catch. That should've been `4.16-9.4` as per the `--name 4.16-9.4` argument.